akka-edge-docs/src/main/paradox/guide/6-deploying-local-drone-control-service.md

```diff
@@ -0,0 +1,5 @@
+# Deploying the Local Drone Control Service
+
+FIXME Outline:
+
+ * the big unknown, or is it just docker k8s (k3s)?
```

Comment:
Not sure, we could just deploy it to Kubernetes to keep things simple?

@pvlugter will write something about scaling, native image and such. Not sure if that should be a separate guide/documentation or if it will continue from here?

Reply:
Keep it simple as docker and lightweight kubernetes sounds good to me. And then the low-resource and scale to/from near-zero as an additional/optional deployment approach on top of this.